clang/test/CodeGen/aarch64-sme-func-attrs-inline/sme-inline-new-za.c

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang --target=aarch64-none-linux-gnu -march=armv9-a+sme -O3 -S -Xclang -verify %s
```

Comment:
Can you merge all these tests into a single file?

Reply:
Done. I originally didn't as the frontend bails out once it's printed all the errors for one function (-ferror-limit doesn't help here), but using macros to only test one caller function at a time mitigated this.